symbolic-debuginfo/tests/fixtures/c8374b6d-6e96-34d8-ae38-efaa5fec424f.bcsymbolmap

```diff
@@ -0,0 +1,3660 @@
+-[SentryEvent transaction]
+-[SentryEvent setTransaction:]
+-[SentryEvent type]
+-[SentryEvent setType:]
+-[SentryEvent tags]
+-[SentryEvent setTags:]
+-[SentryEvent extra]
+-[SentryEvent setExtra:]
+-[SentryEvent sdk]
+-[SentryEvent setSdk:]
+-[SentryEvent modules]
+-[SentryEvent setModules:]
+-[SentryEvent fingerprint]
+-[SentryEvent setFingerprint:]
+-[SentryEvent user]
+-[SentryEvent setUser:]
+-[SentryEvent context]
+-[SentryEvent setContext:]
+-[SentryEvent threads]
+-[SentryEvent setThreads:]
+-[SentryEvent exceptions]
+-[SentryEvent setExceptions:]
+-[SentryEvent stacktrace]
+-[SentryEvent setStacktrace:]
+-[SentryEvent debugMeta]
+-[SentryEvent setDebugMeta:]
+-[SentryEvent breadcrumbs]
+-[SentryEvent setBreadcrumbs:]
+-[SentryEvent .cxx_destruct]
+_OBJC_CLASSLIST_REFERENCES_$_.42
+__OBJC_CLASS_PROTOCOLS_$_SentryEvent
+__OBJC_METACLASS_RO_$_SentryEvent
+__OBJC_$_INSTANCE_METHODS_SentryEvent
+_OBJC_IVAR_$_SentryEvent._eventId
+_OBJC_IVAR_$_SentryEvent._message
+_OBJC_IVAR_$_SentryEvent._error
+_OBJC_IVAR_$_SentryEvent._timestamp
+_OBJC_IVAR_$_SentryEvent._startTimestamp
+_OBJC_IVAR_$_SentryEvent._level
+_OBJC_IVAR_$_SentryEvent._platform
+_OBJC_IVAR_$_SentryEvent._logger
+_OBJC_IVAR_$_SentryEvent._serverName
+_OBJC_IVAR_$_SentryEvent._releaseName
+_OBJC_IVAR_$_SentryEvent._dist
+_OBJC_IVAR_$_SentryEvent._environment
+_OBJC_IVAR_$_SentryEvent._transaction
+_OBJC_IVAR_$_SentryEvent._type
+_OBJC_IVAR_$_SentryEvent._tags
+_OBJC_IVAR_$_SentryEvent._extra
+_OBJC_IVAR_$_SentryEvent._sdk
+_OBJC_IVAR_$_SentryEvent._modules
+_OBJC_IVAR_$_SentryEvent._fingerprint
+_OBJC_IVAR_$_SentryEvent._user
+_OBJC_IVAR_$_SentryEvent._context
+_OBJC_IVAR_$_SentryEvent._threads
+_OBJC_IVAR_$_SentryEvent._exceptions
+_OBJC_IVAR_$_SentryEvent._stacktrace
+_OBJC_IVAR_$_SentryEvent._debugMeta
+_OBJC_IVAR_$_SentryEvent._breadcrumbs
+__OBJC_$_INSTANCE_VARIABLES_SentryEvent
+__OBJC_$_PROP_LIST_SentryEvent
+__OBJC_CLASS_RO_$_SentryEvent
+/Users/philipphofmann/git-repos/sentry-cocoa/Sources/Sentry/SentryEvent.m
+Sources/Sentry/SentryEvent.m
+Sources/Sentry/Public/SentryEvent.h
+__OBJC_METACLASS_RO_$_SentryCrashExceptionApplication
+__OBJC_CLASS_RO_$_SentryCrashExceptionApplication
+/Users/philipphofmann/git-repos/sentry-cocoa/Sources/Sentry/SentryCrashExceptionApplication.m
+_g_fatalSignalData
+_g_fatalSignals
+_g_sigBusCodes
+_g_sigFPECodes
+_g_sigIllCodes
+_g_sigSegVCodes
+_g_sigTrapCodes
+/Users/philipphofmann/git-repos/sentry-cocoa/Sources/SentryCrash/Recording/Tools/SentryCrashSignalInfo.c
+sentrycrashsignal_numFatalSignals
+Sources/SentryCrash/Recording/Tools/SentryCrashSignalInfo.c
+sentrycrashsignal_fatalSignals
+sentrycrashsignal_signalCodeName
+sentrycrashsignal_signalName
+_handleSignal
+_sentrycrashcm_signal_getAPI.api
+_g_eventID
+_g_signalStack
+_g_previousSignalHandlers
+/Users/philipphofmann/git-repos/sentry-cocoa/Sources/SentryCrash/Recording/Monitors/SentryCrashMonitor_Signal.c
+Sources/SentryCrash/Recording/Monitors/SentryCrashMonitor_Signal.c
```

Comment:
I’m a bit surprised to find (absolute) filenames here. Are those being obfuscated as well? Do we need to resolve them when we extract file/line info?

Reply:
huh, intersting.  i must admit i did not check this file...  Yes, I imagine that this is also obfuscated. I guess question is whether we need to de-obfuscate other things than just symbols as well I guess.  We should look at this sometime.